airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/util/ConnectorExceptionUtil.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.util
+
```

Comment:
old CDK's ConnectorExceptionUtil had a ton of other stuff in it. I just copied in the thing we're using.